app/containers/MessageBox/EmojiPicker/categories.js

```diff
@@ -0,0 +1,36 @@
+const list = ['Smileys & People', 'Animals & Nature', 'Food & Drink', 'Activities', 'Travel & Places', 'Objects', 'Symbols', 'Flags'];
+const tabs = [
```

Comment:
@diegolmello I think we should also use **recently used** tab in this.

Reply:
Read PR todo list